docs/assert/false.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: default
+title: false
+description: A boolean check. Passes if the first argument is false.
+categories:
+  - assert
+---
+
+## `false( state [, message ] )`
+
+A boolean check, inverse of `true()` equivalent to Chais's `assert.isFalse()`, and JUnit's `assertFalse()`. Passes if the first argument is false.
+
+| name               | description                          |
+|--------------------|--------------------------------------|
+| state              | Expression being tested              |
+| message (string)   | A short description of the assertion |
+
+### Description
+
+`false()` requires just one argument. If the argument evaluates to false, the assertion passes; otherwise, it fails. If a second message argument is provided, it will be displayed in place of the result.
+
+### Examples:
```

Comment:
We usually have our sections without colon.

Reply:
My bad: this is coming from copy-pasting the [`notOk` docs](https://api.qunitjs.com/assert/notOk). We should make sure to fix that as part of https://github.com/qunitjs/qunit/issues/1450. I also added the (correct) header to `true.md` which was missing (as it's missing in [`ok` docs](https://api.qunitjs.com/assert/ok)).